### ISSUE TITLE
Game Fails To Update Via Client "Content" Update Button

### ISSUE THREAD

**Issue by TechnoWolfTV** [minetest/minetest#14682]
### Minetest version

```true
Minetest 5.8.0 Win64
```

### Operating system and version

Windows 11 Home 23H2

### Summary

I'm the maintainer of a game on ContentDB titled, "Adventurelands". I have noticed that when attempting to update the game via client it fails every time and on any device I attempt to do so on. No other mods or games seem to have this problem. ContentDB suggested posting a bug report here. After an update attempt, the following error is displayed:

> Error installing "Adventurelands": Failed to install adventurelands to C:\Users\technowolftv\Documents\Minetest\games\adventurelands

After this, the game is missing from the main menu. When browsing to the install path, the game's folder is still there but the game.conf is missing (and possibly some other files) which explains why client cannot see it any more. When examining debug.txt, there are a number of errors logged about this. Example:

> 2024-05-14 10:10:43: ERROR[Main]: RecursiveDelete: Failed to delete file C:\Users\technowolftv\Documents\Minetest\games\adventurelands\menu\theme.ogg
> 2024-05-14 10:10:43: ERROR[Main]: RecursiveDelete: Failed to recurse to C:\Users\technowolftv\Documents\Minetest\games\adventurelands\menu\theme.ogg
> 2024-05-14 10:10:43: ERROR[Main]: RecursiveDelete: Failed to recurse to C:\Users\technowolftv\Documents\Minetest\games\adventurelands\menu
> 2024-05-14 10:10:43: ERROR[Main]: MoveDir: target "C:\Users\technowolftv\Documents\Minetest\games\adventurelands" exists as file or non-empty folder
> 2024-05-14 10:12:54: ERROR[Main]: RecursiveDelete: Failed to delete file C:\Users\technowolftv\Documents\Minetest\games\adventurelands

So, in order to update, I must manually delete the adventurelands folder at C:\Users\technowolftv\Documents\Minetest\games\adventurelands and then reinstall from scratch.

### Steps to reproduce

Have an existing install of the "Adventurelands" game installed via client, and then attempt to update it via client when an update is available.

**Comment by grorp** [comment]
duplicate of #14193?

**Comment by Laudrin** [comment]
I had the same problem with X Farming mod and another one where I don't remember the name. It seems to be no single-mod-problem, but something which could happen with some or even all mods.

My solution was to delete the mod folder by hand and then reinstall the mod.

> duplicate of https://github.com/minetest/minetest/issues/14193?

Not sure about the OP, but In my case this was with a freshly started MT. When this error occurred the first time, I did restart MT and then ignored the update for a few days, in which I'm sure I rebooted the computer at least one time. So it should be no error of Windows locking a used file imho, at least if MT does not use any mod files at startup, before any game / world is loaded.
